core/src/main/res/values/strings.xml

```diff
@@ -506,8 +506,8 @@
     <string name="pref_cast_message_free_flavor">Chromecast requires third party proprietary libraries that are disabled in this version of AntennaPod</string>
     <string name="pref_enqueue_downloaded_title">Enqueue Downloaded</string>
     <string name="pref_enqueue_downloaded_summary">Add downloaded episodes to the queue</string>
-    <string name="media_player_builtin">Built-in Android player</string>
-    <string name="media_player_sonic" translatable="false">Sonic Media Player</string>
+    <string name="media_player_builtin">Built-in Android player (unsupported) </string>
+    <string name="media_player_sonic" translatable="false">Sonic Media Player (unsupported) </string>
```

Comment:
It's currently set to `translatable="false"`. How about making it translatable?

How about calling it "deprecated" instead? I am not a native speaker but to me it sounds a bit like AntennaPod can not handle the media players, while it should actually mean that we no longer develop them.

Reply:
fixed